packages/wrangler/src/config.ts

```diff
@@ -41,6 +41,12 @@ type DurableObject = {
   script_name?: string;
 };
 
+type Service = {
+  name?: string;
+  script_name: string;
```

Comment:
What about this one here @Electroid? While I agree that we shouldn't use `script_name` anymore, we use it for consistency with Durable Objects. Should we change `CfServiceRef` in the other file , but not `Service` here?

Reply:
Note this one is what defines what we put in `wrangler.toml`, the definition at `src/api/worker.ts` is our internal representation once parsed.

Reply:
I think we should use `script_name`. `name` is too generic, and in this case it's confusing whether it's self referential. And because we've already used script_name for a similar metaphor for durable objects, we should probably stick to it